internal/assertions/assertions.go

```diff
@@ -13,6 +13,7 @@ import (
 	"strings"
 
 	"github.com/google/go-cmp/cmp"
+
```

Comment:
@pkazmierczak do you mind clearing the extra added newline?

Reply:
oof sorry I think it's my goland settings that did this.